compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/Compiler.java

```diff
@@ -105,11 +105,11 @@ public void compile(String sourcePkg) {
         }
 
         pkgNode = codeAnalyze(pkgNode);
-        if (this.stopCompilation(CompilerPhase.TAINT_ANALYZE)) {
-            return;
-        }
-
-        pkgNode = taintAnalyze(pkgNode);
+//        if (this.stopCompilation(CompilerPhase.TAINT_ANALYZE)) {
```

Comment:
Removed the taint analyzer temporarily. @ayomawdb  Please fix. Thanks

Reply:
@sameera-jayasoma Noted. Please add me in the PR with Supun's changes as well.